### PR TITLE
Done with the signup and login forms

### PR DESCRIPTION
**Pull request by sohan9819**
Completed the login and signup forms 

image

image

click on the login button to navigate to the login/signup page

### CODE REVIEW

**Review by Rutwik87 — APPROVED** [review]
Looks Fine

**Comment by rahul31agrawal** [comment]
Hi Sohan,
The login & signup form looks good to me
looks good to merge

**Review by rahul31agrawal — APPROVED** [review]

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            </div>
+
+            <div class="input__text-group">
+              <input
+                type="password"
+                class="input__text-input box-shadow"
+                placeholder="Password"
+                id="email"
+                required
+                pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
```
Comment (shubhmehra) [inline pages/login.html:166]: your id is for email but you are using password type!
that's confusing! please check!

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+            <div class="signup__form__options h4 mb-2">
+              <label for="remember">
+                <input type="checkbox" name="remember" id="remember" required />
+                I accept all tems and conditions
+              </label>
+
+              <a href="#" class="mr-2">forgot your password</a>
+            </div>
+
```
Comment (shubhmehra) [inline pages/login.html:186]: you can use prettier for this blank spaces

**Review by shubhmehra — APPROVED** [review]
approved though...
some comments added please check them before merging..

File: pages/login.html (hunk 1 above)
Comment (sudosohan) [reply pages/login.html:166]: Yeah thanks for pointing that I fixed it 👍

File: pages/login.html (hunk 2 above)
Comment (sudosohan) [reply pages/login.html:186]: Yeah I am already using it

**Pull request merged**